ui/src/worker/jobs/fetchFluxData.ts

```diff
@@ -17,7 +18,9 @@ export default async (msg: Message): Promise<DecodeFluxRespWithLimitResult> => {
     uuid,
   })
 
-  const response = await fetch(url, {
+  let response
```

Comment:
I couldn't figure out the reason you used a let here, it doesn't look like it's being reassigned?

Reply:
ah i had moved the try up to test something and needed a let so it could be defined outside the try. ill change that back.